app/waf/src/models/service.model.ts

```diff
@@ -1,50 +1,344 @@
-import {Entity, model, property} from '@loopback/repository';
+import {model, property} from '@loopback/repository';
+import {CommonEntity} from '.';
 
 @model()
-export class Service extends Entity {
+export class Service extends CommonEntity {
   @property({
     type: 'string',
-    id: true,
+    required: true,
+    schema: {
+      create: true,
+      response: true,
+      required: true,
+      example: 'HTTP',
+    },
+  })
+  type: string;
+
+  @property({
+    type: 'string',
+    required: true,
+    schema: {
+      create: true,
+      required: true,
+      response: true,
+      example: '11111111-2222-3333-4444-555555555555',
+    },
+  })
+  applicationId: string;
+
+  @property({
+    type: 'boolean',
+    required: false,
+    default: true,
+  })
+  addressStatus: boolean;
```

Comment:
Will add the schemas for these properties in the future?

Reply:
The properties without schema definition is not supported by our API now. They are excluded from response and will be disabled by the validator (todo). Will add schema, once we support those properties.

Reply:
ok